chacractl/util.py

```diff
@@ -20,6 +21,7 @@ def load_config():
     config = get_config_path()
     if not config:
         return
+    sys.modules.pop('chacractl', None)
```

Comment:
Originally I did implement this as a test fixture, but then I had to explicitly use the fixture in *every* test that happens to call `load_config()`. Otherwise the tests could be subject to ordering failures or false-positives.

The fact that `load_config()`'s return value depends on global state like this pointed me in the direction of just fixing it here in `load_config()`.